frontends/ol-util/src/hooks/index.ts

```diff
@@ -1,3 +1,2 @@
 export { default as useSearchParams } from "./useSearchParams"
-export { default as useId } from "./useId"
```

Comment:
`useId` is builtin to react 18; we had back-ported it, but now that we're on React 18, I removed it.